content/documentation/getting-started/index.adoc

```diff
@@ -63,51 +66,88 @@ If the Kiali version packaged with Istio or Maistra meets your needs then instal
 
-icon:lightbulb[size=1x]{nbsp} Environment variables or command-line options affect the Kiali Operator install script.  For example, if you do not want the script to install a Kiali CR (and thus avoid having the Kiali Operator install Kiali itself), set the OPERATOR_INSTALL_KIALI environment variable to "false" and then run the script.  Or run the script with "-oik=false" Read the comments at the top of the link:https://github.com/kiali/kiali/blob/master/operator/deploy/deploy-kiali-operator.sh[Kiali Operator install script] for details on what custom configuration settings are available.  Alternatively, use the script's "--help" command line option for help documentation.
+==== Demo/Dev Installation
 
-icon:bullhorn[size=1x]{nbsp} The install script requires `envsubst` installed and in your PATH; you can get it via the GNU `gettext` package.
+Like the Default Installation but Kiali will be able to access existing namespaces and future namespaces. Because this grants cluster roles it is not recommended for production. Run the install script via this command:
+
+[source,bash]
+----
+bash <(curl -L https://git.io/getLatestKialiOperator) --accessible-namespaces
+----
 
-During the installation, you will be prompted for the authentication strategy you want to use among `login`, `anonymous` and `openshift`. See the link:#_login_options[Login Options] section below for more information.
 
-=== Configure Kiali
+==== Custom Installation
 
-The Kiali Operator runs inside the `kiali-operator` namespace and listens for a Kiali CR (custom resource) creation or change. On creation it will install Kiali using the settings it finds in the CR object. To see all the different settings you can configure in a Kiali CR, see link:https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[this fully documented example Kiali CR yaml file].
+This option installs the Kiali Operator and a Kiali CR (Custom Resource) with a custom configuration. To generate the CR file it is recommended to copy the fully documented link:https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[example Kiali CR yaml file].  Edit the file, being careful to maintain proper formatting, and save it to a local file (e.g. my-kiali-cr.yaml).
 
 icon:lightbulb[size=1x]{nbsp} It is important to understand the `deployment:accessible_namespaces` setting in the CR.  See link:#_accessible_namespaces[Accessible Namespaces] for more information.
 
-To edit from a terminal, run:
+Run the install script via this command (replacing the CR filename as needed):
+
+[source,bash]
+----
+bash <(curl -L https://git.io/getLatestKialiOperator) --kiali-cr my-kiali-cr.yaml
+----
+
+
+==== Post Install
+
+After installation is complete using one of the above options you are free to update your Kiali CR.  The Kiali Operator will automatically apply any updates.  To edit the Kiali CR run the command (note: the default kiali CR name is `kiali`):
```

Comment:
Maintain consistent capitalization. Everywhere else its called `Kiali CR` (not kiali CR)
```suggestion
After installation is complete using one of the above options you are free to update your Kiali CR.  The Kiali Operator will automatically apply any updates.  To edit the Kiali CR run the command (note: the default Kiali CR name is `kiali`):
```

Reply:
+1 fixed in multiple places